tests/greenheart/test_greensteel/input/floris/floris_input_lbw_6MW.yaml

```diff
@@ -0,0 +1,76 @@
+name: Gauss
```

Comment:
This input file is missing the farm section, defining the layout and the turbine type. Is that correct? I don't think FLORIS will run without adding that info.

Reply:
GreenHEART overwrites it for now. We would like to have it both ways eventually, but the ORBIT custom layout has not worked for me yet so right now we just use a grid in GreenHEART.

Reply:
So, for GreenHEART we don't have to have that section in the input files.